crates/types/src/config/ingress.rs

```diff
@@ -45,6 +45,10 @@ pub struct IngressOptions {
     /// back to a previous version.
     #[cfg_attr(feature = "schemars", schemars(skip))]
     pub experimental_feature_enable_separate_ingress_role: bool,
+
+    /// Cluster of new features for the kafka ingress.
```

Comment:
Are you gonna expand on what this cluster of new features are as you are tackling the other issues?

Reply:
I put it in the commit. I won't put them in the code until i get them done :smile: Also check https://github.com/orgs/restatedev/projects/15/views/13?sliceBy%5Bvalue%5D=ingress_kafka